### DISCUSSION TITLE
Proj file adds `<None Remove>`, `<Page Update>`, ..., every time a new xaml file is added

### DISCUSSION THREAD

**Discussion by karmeye** [microsoft/microsoft-ui-xaml#10013] [Q&A]
Are the below things that are added automatically to the proj file needed? Seems to work fine without them. What are they for? The proj file gets pretty bloated with these.

```xaml
<ItemGroup>
   <None Remove="Controls\DeveloperControl.xaml" />
</ItemGroup>

<ItemGroup>
   <Page Update="Controls\DeveloperControl.xaml">
      <Generator>MSBuild:Compile</Generator>
   </Page>
</ItemGroup>
```

Thanx

**Comment by whiskhub** [comment]
I believe they are a relict of the past, when the old project file format had no automatic file globbing.
I think you could open a feature request in the repo for this.
